tests/system/Database/Live/ForgeTest.php

```diff
@@ -280,4 +280,43 @@ public function testDropForeignKey()
 		$this->forge->dropTable('forge_test_users', true);
 
 	}
+
+	public function testEnumSetFields()
+	{
+		if ($this->db->DBDriver !== 'MySQLi')
```

Comment:
We need to test Postgres, also. And SQLite is coming fairly soon....

Reply:
As I understand, not all databases have `ENUM` data type and `SET` is more specific in MySQL.

In the case of SQLite, it does not have any of these two types.

I thought about separating the test of each type into methods and then go updating the list of valid databases as they arrive.